deploy/chart/values.yaml

```diff
@@ -40,6 +40,7 @@ catalog:
   setWorkloadUserID: true
   replicaCount: 1
   commandArgs: --configmapServerImage=quay.io/operator-framework/configmap-operator-registry:latest
+  opmImageArgs: --opmImage=quay.io/operator-framework/opm:latest
```

Comment:
I'm not very familiar with this code base, could you provide any resources about how this is injecting this image as the opm image to use?

Reply:
It's [this](https://github.com/operator-framework/operator-lifecycle-manager/pull/3275/commits/af279d96d68467c3808fc33e6aaaa4b5b7632ae0#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R257) bit that does the injection. This just sets a reasonable default for the value.

Reply:
Apologies, I think I could have phrased this better - Where is this `--opmImage` flag defined and where is the provided value used within OLM?

By no means should this comment block this PR, but since I'm still relatively new to the codebase the additional context would be helpful for me :)

Reply:
All good, man. I'm learning myself 😂

This flag is defined by the catalog operator binary (in the cmd package)